README.md

```diff
@@ -28,6 +28,10 @@ This Helm chart installs and configures the following operators:
 
 **Note**: If a subscription for an operator already exists, the installation will not tamper with it.
 
+### Helm Troubleshooting
```

Comment:
With my other comment might make sense to change the ordering to be between **uninstall** and **configuration** sections.